Code/tests/test_expand_repo.py

```diff
@@ -12,13 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import imp
-import os
 import unittest
 
-autopkg = imp.load_source(
-    "autopkg", os.path.join(os.path.dirname(__file__), "..", "autopkg")
-)
+from autopkgcli import autopkg
```

Comment:
Same issue: I think this will have an issue if you are trying to dev/test autopkg source, then this will actually load autopkg from the pip installed version, not from the source version you are currently testing as a cloned repo.

I think this should only be a problem if you have previously pip installed autopkg? or maybe it only works if you have pip installed autopkg and won't work otherwise? I don't remember exactly.

Reply:
Not if you start using env python as your dev python

Reply:
If this get merged, additional PRs will obviously need to be done to update docs on install and dev environment setup.

Reply:
well I don't think using this method is required especially for the default use case on MacOS in which case the PKG would still be preferred.